databricks/koalas/groupby.py

```diff
@@ -211,6 +211,10 @@ def aggregate(self, func_or_funcs=None, *args, **kwargs):
             kdf = kdf.reset_index(drop=self._should_drop_index)
 
         if relabeling:
+
+            # For MultiIndex, we need to flatten the tuple, e.g. (('y', 'A'), 'max') needs to be
+            # flattened to ('y', 'A', 'max'), it won't do anything on normal Index.
+            order = [(*levs, method) for levs, method in order]
```

Comment:
@itholic, can we fix it in `_normalize_keyword_aggregation`?

Reply:
Got it! 👍